po/list_translators

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -e -u
+
+SCRIPT_NAME=$(basename "$0")
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
+source "$MB_SERVER_ROOT"/script/macos_compat.inc.sh
+
+declare -a OMITTED_AUTHORS=(
+  # From the add-on "Update PO files to match POT (msgmerge)"
+  'Hosted Weblate'
+  # Contractors, avoiding to be self-congratulatory
+  bitmap
+  reosarevok
+  yvanzo
+)
+
+GIT_REVISION_RANGE='origin/production..origin/beta'
+
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME [<git-revision-range>]
+
+List the translators for the given Git revision range.
+
+The default Git revision range is $GIT_REVISION_RANGE
+assuming that this script is called before merging to production.
+
+The following translation authors are intentionally unlisted:
+    $(printf '%s, ' "${OMITTED_AUTHORS[@]}" | sed "s_, \$__")
+
+Commas (and surrounding spaces) in author names are replaced with slashes (/).
+EOH
+)
+
+if [[ $# -gt 1 ]]
+then
+  echo >&2 "$SCRIPT_NAME: too many arguments"
+  echo >&2 "$HELP"
+  exit 64
+elif [[ $# -eq 1 ]]
+then
+  if echo "$1" | grep -Eqx -- '-*h(elp)?'
+  then
+    echo "$HELP"
+    exit
+  else
+    GIT_REVISION_RANGE=$1
+  fi
+fi
+
+cd "$MB_SERVER_ROOT"
+
+git log "$GIT_REVISION_RANGE" -- 'po/*.po' | \
+  # Capture translation author full names
+  sed -n 's/^Author: \(.*\) <.*$/\1/p' | \
+  # Omit Weblate authors and blog post authors
+  grep -iv "^\($(printf '%s\|' "${OMITTED_AUTHORS[@]}" | sed 's/\|$//')\)$" | \
```

Comment:
Adding the extra backslash there seems to fix things for me, can you test it on your machine to be sure? :)
```suggestion
  grep -iv "^\($(printf '%s\|' "${OMITTED_AUTHORS[@]}" | sed 's/\\|$//')\)$" | \
```

Reply:
Tested locally, both variants do work on Linux. Should now be fixed. See https://github.com/metabrainz/musicbrainz-server/compare/79396b6b2ac9f3ce7b260355030fa7a0a79a1d22..7c53cecd5cfde756115e802222b7a7b5618d5f3d